applications/StructuralMechanicsApplication/tests/test_adjoint_loading_conditions.py

```diff
@@ -4,14 +4,14 @@
 import KratosMultiphysics.KratosUnittest as KratosUnittest
 import math
 
-class TestAdjointLoadingConditions(KratosUnittest.TestCase):
+def GetZeroMatrix(num_lines, num_columns):
+    zero_matrix = KratosMultiphysics.Matrix(num_lines, num_columns)
```

Comment:
@philbucher is their already a possibility to get a zero kratos matrix in python? I was not able to find it.